src/dialog/overflow-menu/overflow-menu.stories.ts

```diff
@@ -0,0 +1,33 @@
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+
+import { TranslateModule } from "@ngx-translate/core";
+
+import { DialogModule } from "../../";
+
+storiesOf("Overflow Menu", module)
+	.addDecorator(
+		moduleMetadata({
+			imports: [
+				DialogModule,
+				TranslateModule.forRoot()
+			],
+		})
+	)
+	.add("Basic", () => ({
+		template: `
+			<ibm-overflow-menu [options]="overflowContent"></ibm-overflow-menu>
```

Comment:
Brain teaser - what would we have to do to have an option of writing it as:

```html
<ibm-overflow-menu>
		<ibm-overflow-menu-option>
			An example option that is really long to show what should be done to handle long text
		</ibm-overflow-menu-option>
		<ibm-overflow-menu-option>Option 2</ibm-overflow-menu-option>
		<li class="bx--overflow-menu-options__option">
			<button class="bx--overflow-menu-options__btn">A fully custom option</button>
		</li>
		<ibm-overflow-menu-option>Option 4</ibm-overflow-menu-option>
		<ibm-overflow-menu-option disabled="true">Disabled</ibm-overflow-menu-option>
		<ibm-overflow-menu-option type="danger">Danger option</ibm-overflow-menu-option>
</ibm-overflow-menu>
```

Reply:
I just wrote a whole thing about how I couldn't find a way to do that ... Then I figured out a way to do that. 

And it was so stupidly obvious I'm kicking myself for not starting there 🤦‍♂️